__tests__/fixtures/lowercase_link_reference/CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.2.0] - 2019-09-13
+
+### Added
+
+-   First feature that is gonna make us money.
+-   Quite a few bugs, sorry in advance!
+
+### Changed
+
+-   Reworked the login system. You have to provide a password now!
+
+## [0.1.0] - 2019-09-05
+
+### Added
+
+-   Initial release :tada:
+
+[unreleased]: https://github.com/foo/bar/compare/0.2.0...HEAD
```

Comment:
The only difference to the "standard" test is the lower cased "u" from the word "unreleased".